smoke-test/tests/cypress/integration_test.py

```diff
@@ -99,48 +100,84 @@
     ONBOARDING_IDS.extend(id_list)
 
 
+def print_now():
+    print(f"current time is {datetime.datetime.now(datetime.timezone.utc)}")
+
+
 def ingest_data():
+    print_now()
     print("creating onboarding data file")
     create_datahub_step_state_aspects(
         get_admin_username(),
         ONBOARDING_IDS,
         f"{CYPRESS_TEST_DATA_DIR}/{TEST_ONBOARDING_DATA_FILENAME}",
     )
 
+    print_now()
     print("ingesting test data")
     ingest_file_via_rest(f"{CYPRESS_TEST_DATA_DIR}/{TEST_DATA_FILENAME}")
     ingest_file_via_rest(f"{CYPRESS_TEST_DATA_DIR}/{TEST_DBT_DATA_FILENAME}")
     ingest_file_via_rest(f"{CYPRESS_TEST_DATA_DIR}/{TEST_SCHEMA_BLAME_DATA_FILENAME}")
     ingest_file_via_rest(f"{CYPRESS_TEST_DATA_DIR}/{TEST_ONBOARDING_DATA_FILENAME}")
+    print_now()
+    print("completed ingesting test data")
 
 
 @pytest.fixture(scope="module", autouse=True)
 def ingest_cleanup_data():
     ingest_data()
     yield
+    print_now()
```

Comment:
Do we still need these debug prints?

Reply:
Yes we should keep these prints so we know where exactly the time is being spent.